### ISSUE TITLE
Choosing a cotton cultivar to clone

### ISSUE THREAD

**Issue by edvlmaas** [APSIMInitiative/APSIMClassic#2198]
Hi, I need to create a new cotton cultivar, which I've done following the steps here: https://github.com/APSIMInitiative/APSIMClassic/issues/2031

It looks like the default cultivar is S71BR, which I assume is what is being cloned when a new cultivar is created. The name of the new cultivar is "_Default_S71BR", and the cultivar_default parameter is indeed set to "S71BR". 

However, S324 is the closest fit (R^2 and RMSE) to my data, so I would like to clone it and just adjust the yield down. I tried to change the default cultivar by changing the cultivar_default parameter value to "S324", which it accepts, but the value under the "default" column still says S71BR. When I create a new cultivar, the name is still "_Default_S71BR", and when I run the simulation, the results are the same as S71BR, not S324.

Here is a screenshot: 
<img width="461" alt="image" src="https://user-images.githubusercontent.com/15112298/232253942-95ad2831-841b-4b32-9a57-e51fa6980881.png">

So is there a way to clone a cultivar other than S71BR?

**Comment by peter-devoil** [comment]
I'm not 100% up on this part of the GUI, but I think "cultivar_default" is intended to be used by the infrastructure (to generate documentation mainly), not by users. Users select a cultivar (eg S324, S71 etc) to be sown via the sowing component - some manager code that can be responsive to seasonal conditions/paddock history/market prices ...

So I'd ignore it, and concentrate on the sowing manager component

**Comment by edvlmaas** [comment]
So I can use the sewing component to make a cultivar? My question is how to make a cultivar based on S324. I searched source code to find what parameters are set differently from the S71BR default, but couldn’t find anything. The cotton.xml doesn’t have them.

**Comment by peter-devoil** [comment]
No, the sowing component selects (instantiates?) a cultivar within a simulation

In the tree display, right clicking on the "cultivars" variable opens a menu "Add", which will duplicate the 1st cultivar. Then change the cultivar's variables accordingly

**Comment by edvlmaas** [comment]
OK, so how can I recreate the S324? I am wanting to start with that and play with percent_l, scboll, and BckGndRetn.

**Comment by peter-devoil** [comment]
No idea. You could try asking Michael dot Bange at grdc dot com dot au

**Comment by edvlmaas** [comment]
I’ll do that. Thank you.Sent from my iPhoneOn Apr 16, 2023, at 10:03 PM, Peter de Voil ***@***.***> wrote:﻿
No idea. You could try asking Michael dot Bange at grdc dot com dot au

**Comment by edvlmaas** [comment]
His email bounced back. I emailed apsim at csiro dot au to ask about it and
was told "Michael Bange no longer works for GRDC and has moved on to Cotton
Seed Distributors" and was given a link with phone numbers.
https://csd.net.au/contact However, I'm in the United States and my cell
phone plan doesn't allow me to call internationally. Anything else you can
suggest?

**Comment by peter-devoil** [comment]
I'll have a look - he may have retired.

**Comment by SaeeTS** [comment]
hi @edvlmaas 

Could you find out how we can create a new cotton cultivar?
